src/main/java/com/company/logic/HistoryVacationLogic.java

```diff
@@ -0,0 +1,84 @@
+package com.company.logic;
+
+import com.company.model.DayVacation;
+import com.company.model.Employee;
+import com.company.model.HistoryVacation;
+import com.company.session.Connection;
+import com.company.tools.ConstantData;
+import com.company.util.Bundle;
+import com.company.util.Either;
+import com.company.util.Error;
+import com.company.util.ObjectResponce;
+import com.company.validation.EmployeeGet;
+import com.company.validation.VacationCreate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.ws.rs.core.Response;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.ibatis.session.SqlSession;
+
+/**
+ *
+ * @author santiago.mamani
+ */
+public class HistoryVacationLogic {
+
+    public ObjectResponce sendVacation(long idEmployee, String startDate, String endDate, String reason) {
```

Comment:
Not sure what is the purpose of this method. There is a `rollback` when an exception happens, but I don't see any `commit`. Also, make sure to `rollback` in cases when no exception is caught but you are returning an Error and you have **executed a persistence operation**.